src/SFA.DAS.Commitments.Support.SubSite/Extensions/EnumExtensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.ComponentModel;
+using System.Reflection;
+
+namespace SFA.DAS.Commitments.Support.SubSite.Extentions
+{
+    public static class EnumExtensions
```

Comment:
I'm sure I've seen this code somewhere before...

Reply:
this is in eas, vs, pas, emp-comt, indexer. Should we not make this part of the Shared packaged if it is being used in so many places?